sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/PropertiesUtils.java

```diff
@@ -33,6 +36,7 @@
 public class PropertiesUtils {
 
 	private static final Logger log = LoggerFactory.getLogger(PropertiesUtils.class);
+	private static final String SDK_PROPERTIES_FILE = "SDK Properties file";
```

Comment:
Shouldn't this just be something like `sdk.properties`?

Reply:
When writing the SDK properties to the files using [Properties#store(java.io.OutputStream, java.lang.String)](https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html#store-java.io.OutputStream-java.lang.String-) we have to pass a comment. This comment is used as the heading of the file. I renamed the variable name to eliminate any confusion. The actual file name of the properties file is "sdk.properties".

Reply:
Then the constant name would be less confusing if it reflected that in one way or another like SDK_PROPERTIES_FILE_HEADER. Or just get rid of the constant because after all, you use it in only one place.

Reply:
@dkayiwa I removed the variable in this PR: https://github.com/openmrs/openmrs-sdk/pull/257